src/routes/Patient/routes/Demographics/Demographics.jsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { TitledCard, Grid, Box, Icon, Label, Stack, Button } from 'components/ui/Core.jsx';
+import { usePatient } from 'components/contexts/PatientContext.jsx';
+
+const Demographics = () => {
+  const { useChart, useEncounter } = usePatient();
+  const [{
+    id,
+    firstName,
+    lastName,
+    birthdate,
+    address,
+    preferredLanguage,
+    gender
+  }] = useChart()();
+  const [socioeconomicData] = useEncounter().history.Socioeconomic();
+  const demographics = socioeconomicData?.demographics;
```

Comment:
Technically this line should be `const [demographics, setDemographics] = useEncounter().history.Socioeconomic.demographics()` - unless you also need to access `socioeconomicData` for something else, **_but..._**

Let's actually make this a separate object on the patient chart instead of tied to the encounter. You would be able to access it using `useChart().demographics` once that's modified.  I can do that once this PR is merged so there's nothing for you to change on that front just yet.